src/ZoneServer/Skills/Handlers/Swordsman/Hoplite/ThrowingSpear.cs

```diff
@@ -0,0 +1,198 @@
+				pad.Position = farPos;
+				pad.Trigger.UpdateInterval = TimeSpan.FromSeconds(2);
+				pad.Trigger.Destroyed += this.SpearExplosion;
+
+				caster.Map.AddPad(pad);
+			}
+		}
+
+		/// <summary>
+		/// Executes the main attack
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="splashArea"></param>
+		private async void Attack(Skill skill, ICombatEntity caster, ISplashArea splashArea)
+		{
+			var damageDelay = TimeSpan.FromMilliseconds(50);
+			var skillHitDelay = TimeSpan.Zero;
+
+			await Task.Delay(hitDelay);
+
+			var targets = caster.Map.GetAttackableEntitiesIn(caster, splashArea);
+			var hits = new List<SkillHitInfo>();
+
+			foreach (var target in targets.LimitBySDR(caster, skill))
+			{
+				var modifier = SkillModifier.MultiHit(3);
+
+				var skillHitResult = SCR_SkillHit(caster, target, skill, modifier);
+
+				if (skillHitResult.Result == HitResultType.Crit && caster.Components.TryGet<SkillComponent>(out var skills)
+					&& skills.TryGet(SkillId.Hoplite_SharpSpear, out var sharpSpear))
+				{
+					skillHitResult.Damage += skillHitResult.Damage *= (0.1f + sharpSpear.Level * 0.02f);
+				}
+
+				target.TakeDamage(skillHitResult.Damage, caster);
+
+				var skillHit = new SkillHitInfo(caster, target, skill, skillHitResult, damageDelay, skillHitDelay);
+
+				if (caster.Components.TryGet<AbilityComponent>(out var abilities) && abilities.IsActive(AbilityId.Hoplite33))
+				{
+					skillHit.HitEffect = HitEffect.Impact;					
+				} else
+				{
+					skillHit.KnockBackInfo = new KnockBackInfo(caster.Position, target.Position, skill);
+					skillHit.HitInfo.Type = skill.Data.KnockDownHitType;
+					target.Position = skillHit.KnockBackInfo.ToPosition;
+				}
+
+				target.StartBuff(BuffId.Common_Shock, skill.Level, 0, DebuffDuration, caster);
+				
+				hits.Add(skillHit);
+			}
+
+			Send.ZC_SKILL_HIT_INFO(caster, hits);
+		}
+
+
+		/// <summary>
+		/// Called by the spear explosion effect
+		/// </summary>
+		/// <param name="sender"></param>
+		/// <param name="args"></param>
+		private void SpearExplosion(object sender, TriggerArgs args)
+		{
+			if (args.Trigger is not Pad pad) return;
+			if (pad.Creator is not ICombatEntity creator) return;
+			if (pad.Skill is not Skill skill) return;
+
+			this.ExplosionAttack(pad.Skill, creator, (ISplashArea)pad.Area);
+		}
+
+		/// <summary>
+		/// Executes the secondary attack from the spear explosion
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="splashArea"></param>
+		private void ExplosionAttack(Skill skill, ICombatEntity caster, ISplashArea splashArea)
+		{
+			var damageDelay = TimeSpan.FromMilliseconds(50);
+			var skillHitDelay = TimeSpan.Zero;
+
+			var targets = caster.Map.GetAttackableEntitiesIn(caster, splashArea);
+			var hits = new List<SkillHitInfo>();
+
+			// The explosion has its own SR which is separate from the skill
+			var sr = 8;
```

Comment:
I think you should stop calling this S(D)R, because that's not what it is. Also, if this is an inherent feature we need for pads, to retrieve the amount of actors limited by concurrent use, we should add that instead.

Reply:
I'm confused, to me this does seem to be SDR.  Isn't SDR the maximum target count of a skill?

Reply:
Not exactly, even though you may view it like that for simplicity's sake. It is our current understanding that a character's/skill's `SR` affects the number of targets it can hit, but the actual limiting factor is the targets' `SDR`, which differs between players and monsters and is affected by things like equipment, buffs, sizes, etc. There is a clear distinction between this and a simple target counter. If you call this `SR`, I'm starting to wonder why you aren't using the target's `SDR`, or why the attacker's actual `SR` isn't taken into consideration. The answer to that, as well as the question why the number 8 you have presumably found in the pad_skill_list doesn't match the skill's `SR`, is that that's not what his is. It's the "concurrent use" of the pad. Something separate from the skill's `SR`, as you rightfully concluded.

Reply:
I understand what you're getting at, but I still don't know how we should handle it in this case.